backend/src/main/java/gov/cdc/usds/simplereport/config/FeatureFlagsConfig.java

```diff
@@ -0,0 +1,43 @@
+package gov.cdc.usds.simplereport.config;
+
+import gov.cdc.usds.simplereport.db.model.FeatureFlag;
+import gov.cdc.usds.simplereport.db.repository.FeatureFlagRepository;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties(prefix = "features")
+@EnableScheduling
+@Data
+public class FeatureFlagsConfig {
+  @Autowired
+  @Getter(AccessLevel.NONE)
+  @Setter(AccessLevel.NONE)
+  private FeatureFlagRepository _repo;
+
+  private boolean multiplexEnabled;
+
+  @Scheduled(fixedRateString = "60000") // 1 min
+  private void loadFeatureFlagsFromDB() {
+    Iterable<FeatureFlag> flags = _repo.findAll();
+    flags.forEach(
+        flag -> {
+          flagMapping(flag.getName(), flag.getValue());
+        });
+  }
+
+  private void flagMapping(String flagName, Boolean flagValue) {
+    switch (flagName) {
+      case "multiplexEnabled":
```

Comment:
Does this mean that for future feature flags, we'll need to update the database with the flag and edit this mapping as well?

Reply:
yes, as the flagConfig object and the featureFlag object are not 1 to 1. I think this mapping will be handing when expanding support of feature flags  by facility and org